installation/touchegg.service.in

```diff
@@ -5,7 +5,7 @@ Documentation=https://github.com/JoseExposito/touchegg/tree/master/installation#
 [Service]
 Type=simple
 Group=input
-ExecStart=/usr/bin/touchegg --daemon
+ExecStart=@CMAKE_INSTALL_BINDIR@/touchegg --daemon
```

Comment:
Does this work in your case? On Ubuntu and Fedora the output is `ExecStart=bin/touchegg --daemon`. It looks like it is missing the start `/` and also using `/bin` instead of `/usr/bin`.

I'm not sure why though, the same var works in the `install` command :thinking:

Reply:
Both produced the _same_ path for me, so thats great! Fixed in cab9ef5